app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -37,6 +37,8 @@
 
 public class SettingsFragment extends PreferenceFragment implements SharedPreferences.OnSharedPreferenceChangeListener {
     public static final String SETTINGS_SCREEN_NAME = "settingsScreenName";
+    public static final int ALPHA_DEFAULT = 255;
```

Comment:
I would name these as "ALPHA_ENABLED" and "ALPHA_DISABLED".

Reply:
Agree 👍

Reply:
@layely Please read https://github.com/mozilla-mobile/focus-android/pull/1898#discussion_r154808808 - I don't think setting the alpha directly is necessary and it's a bit more error prone than `setEnabled`.

Reply:
I actually thought that `setEnabled` by itself doesn't work - @layely have you tried the simpler version @mcomella  suggests, and does it work?